### PR TITLE
PixelPaint: Replace vectorscope boxes with colored corner brackets

### PR DESCRIPTION
**Pull request by skmagiik**
This completes a FIXME which requested us to replace the rects/boxes with just corner brackets. I also added some color to the brackets matching the colors they represent. The text remains white :)

### CODE REVIEW

**Comment by kleinesfilmroellchen** [comment]
new vectorscope

File: Userland/Applications/PixelPaint/VectorscopeWidget.cpp [hunk 1]
```diff
@@ -124,11 +124,26 @@ void VectorscopeWidget::paint_event(GUI::PaintEvent& event)
     painter.translate(-static_cast<float>(width()) / 2.0f, -static_cast<float>(height()) / 2.0f);
 
     for (auto const& primary_color : primary_colors) {
-        // FIXME: Only draw the rectangle corners for a more classical oscilloscope look (& less obscuring of color data)
-        auto graticule_rect = Gfx::FloatRect::centered_at(primary_color.to_vector(scope_size), { graticule_size, graticule_size }).to_rounded<int>();
-        base_painter.draw_rect_with_thickness(graticule_rect, graticule_color, graticule_thickness);
-        auto text_rect = graticule_rect.translated(graticule_size / 2, graticule_size / 2);
-        base_painter.draw_text(text_rect, StringView { &primary_color.symbol, 1 }, Gfx::TextAlignment::TopLeft, graticule_color);
+        auto center = primary_color.to_vector(scope_size);
+        auto center_rounded = primary_color.to_vector(scope_size).to_rounded<int>();
```
Comment (kleinesfilmroellchen) [inline Userland/Applications/PixelPaint/VectorscopeWidget.cpp:128]: `center.to_rounded<int>();`

File: Userland/Applications/PixelPaint/VectorscopeWidget.cpp [hunk 2]
```diff
@@ -124,11 +124,26 @@ void VectorscopeWidget::paint_event(GUI::PaintEvent& event)
...
+        // Bottom Left Corner
+        base_painter.draw_line(Gfx::IntPoint(center_rounded.x() - graticule_size / 2, center_rounded.y() + graticule_size / 2), Gfx::IntPoint(center_rounded.x() - graticule_size / 3, center_rounded.y() + graticule_size / 2), corner_color, graticule_thickness);
+        base_painter.draw_line(Gfx::IntPoint(center_rounded.x() - graticule_size / 2, center_rounded.y() + graticule_size / 2), Gfx::IntPoint(center_rounded.x() - graticule_size / 2, center_rounded.y() + graticule_size / 3), corner_color, graticule_thickness);
+        // Bottom Right Corner
+        base_painter.draw_line(Gfx::IntPoint(center_rounded.x() + graticule_size / 2, center_rounded.y() + graticule_size / 2), Gfx::IntPoint(center_rounded.x() + graticule_size / 3, center_rounded.y() + graticule_size / 2), corner_color, graticule_thickness);
+        base_painter.draw_line(Gfx::IntPoint(center_rounded.x() + graticule_size / 2, center_rounded.y() + graticule_size / 2), Gfx::IntPoint(center_rounded.x() + graticule_size / 2, center_rounded.y() + graticule_size / 3), corner_color, graticule_thickness);
```
Comment (kleinesfilmroellchen) [inline Userland/Applications/PixelPaint/VectorscopeWidget.cpp:142]: Can you extract the points to separate variables, as many points are used more than once?

**Review by kleinesfilmroellchen — CHANGES_REQUESTED** [review]
:hyperpoggie:

**Comment by skmagiik** [comment]
@kleinesfilmroellchen Ready for review :)

File: Userland/Applications/PixelPaint/VectorscopeWidget.cpp (hunk 2 above)
Comment (skmagiik) [reply Userland/Applications/PixelPaint/VectorscopeWidget.cpp]: Extracted and updated

File: Userland/Applications/PixelPaint/VectorscopeWidget.cpp (hunk 1 above)
Comment (skmagiik) [reply Userland/Applications/PixelPaint/VectorscopeWidget.cpp]: Updated

**Comment by kleinesfilmroellchen** [comment]
Please squash the two commits into one, and ask on Discord if you need help with that.

**Comment by skmagiik** [comment]
Squashed into one commit

**Review by kleinesfilmroellchen — APPROVED** [review]

**Pull request merged**